### DISCUSSION TITLE
New versioning scheme

### DISCUSSION THREAD

**Discussion by kennethlove** [brack3t/django-braces#284] [Ideas]
`django-braces` is extremely mature and stable. All of the issues it has are related to either explicit version support (Django's usually) or new features.

What if, instead of `django-braces-1.15.0`, we had `django-braces-2.2-3.2-1` to show it was the first release `1`, and for Django versions `2.2` and `3.2`?

I also wouldn't mind a date-based versioning, like I use with `django-shapeshifter`. `django-braces-21.11.15` for example.

Like it? Hate it? Have another versioning scheme that would work better?

Want me to shut up and leave it as it is?

**Comment by auvipy** [comment]
just release django-braces 2.0 O:)

**Comment by kennethlove** [comment]
There have been no breaking API changes. There's no reason for a 2.0.
